nixd/docs/editor-setup.md

```diff
@@ -110,6 +110,11 @@ A simple Emacs Lisp configuration that adds nixd to LSP Mode in the mean time is
                      :server-id 'nixd)))
 ```
 
+> [!NOTE]
+>
+> The client is built-in to lsp-mode after `8.0.0`; you don't need this if you
```

Comment:
> The client

Which client?

> you don't need this if you

Then what should users do (e.g. modifying the config) in this case? Could you provide *a detailed code snippet* to elaborate?

Reply:
> Which client?

The `lsp-mode` is the client implementation. 🤔 

> Then what should users do (e.g. modifying the config) in this case? Could you provide a detailed code snippet to elaborate?

Users should still use the snippet that is already provided.

---

The note simply informs users that they do not need to customize or add a snippet to their configuration after `8.0.0` since it's built-in; otherwise, it is redundant.